index.html

```diff
@@ -0,0 +1,65 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav>
+            <h3 class="logo">My Logo</h3>
+        </nav>
+        
+        <nav>
+            <a href="#">
+            <img src="image/hamburger.png" alt="Hamburger" class="hamburger">
+        </a>
+        </nav>
+       </header>
+
+       <section class="headline">
+        <h1 class="primary">Hey There. <br>I'm Ezra.</h1>
+       
+        <h3 class="supportive">I'm a Software Developer</h3>
+
+        <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+
+        <div class="social-images">
+            <ul class="social">
+                <li class="social-li">
+                    <a href="#">
+                        <img src="image/github.png" alt="Github" class="im-age">
+                    </a>
+                </li>
+                <li class="social-li">
+                    <a href="#">
+                        <img src="image/linkedin.png" alt="LinkedIn" class="im-age">
+                    </a>
+                </li>
+                <li class="social-li">
+                    <a href="#">
+                        <img src="image/thumbs.png" alt="Thumbs" class="im-age">
+                    </a>
+                </li>
+                <li class="social-li">
+                    <a href="#">
+                        <img src="image/twitter.png" alt="Twitter" class="im-age">
+                    </a>
+                </li>
+                <li class="social-li">
+                    <a href="#">
+                        <img src="image/M.png" alt="M" class="im-age">
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="group">
+            <img src="image/Group.png" alt="Group">
+        </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As the above change was not mentioned by the previous reviewer, I think it would be better if you use this image as `background-image` of your section in your `style.css` file and not in a `img` tag. You can that right now it overlaps your paragraph text.
Example:
```css
.headline {
  background-image: url(./image/Group.png);
  background-repeat: non-repeat;
  background-size: 100%;
  ...
  height: 100vh;
  margin-top: 0;
}
```
| Template design | Your Design |
| ------------------------- | -------------------- |
| ![Screenshot from 2023-04-28 00-13-51](https://user-images.githubusercontent.com/83042742/235004997-e5c88d05-5b84-4773-8839-56ebfa5506b9.png) | ![Screenshot from 2023-04-28 00-15-39](https://user-images.githubusercontent.com/83042742/235004820-5edd0c5c-ebf6-48da-a15a-4d88f85c3582.png) |